storage/store/merge.go

```diff
@@ -33,28 +48,28 @@ func (b *baseStore) Merge(kvPartialStore *PartialKV) error {
 	switch b.updatePolicy {
 	case pbsubstreams.Module_KindStore_UPDATE_POLICY_SET:
 		for k, v := range kvPartialStore.kv {
-			b.kv[k] = v
+			b.setKV(k, v)
```

Comment:
`updateKV` to clarify intent, different from `setNewKV` ? Someone could easily mix `setKV` and `setNewKV` not knowing the difference.